test/selenium_tests/test_library_to_collections.py

```diff
@@ -0,0 +1,46 @@
+from .framework import (
+    selenium_test,
+    SharedStateSeleniumTestCase,
+)
+
+
+class LibraryToCollectionsTestCase(SharedStateSeleniumTestCase):
+
+    requires_admin = True
```

Comment:
but it shouldn't ?

Reply:
All the Selenium library test cases do right now - ideally the test cases I guess would create libraries and assign their permissions to another user - but that is a bit more work.